mmv1/products/sql/api.yaml

```diff
@@ -68,6 +68,22 @@ objects:
           - :FIRST_GEN
           - :SECOND_GEN
           - :EXTERNAL
+      - !ruby/object:Api::Type::String
+        name: 'maintenanceVersion'
+        output: true
```

Comment:
Sounds like `maintenanceVersion` is not an output-only field.

Reply:
That's right. It can be updated after instance creation.